projects/plugins/boost/app/lib/critical-css/source-providers/providers/Archive_Provider.php

```diff
@@ -92,13 +92,26 @@ public static function get_post_types() {
 			array(
 				'public'      => true,
 				'has_archive' => true,
-			)
+			),
+			'objects'
 		);
 		unset( $post_types['attachment'] );
 
 		$post_types = array_filter( $post_types, 'is_post_type_viewable' );
 
-		return apply_filters( 'jetpack_boost_critical_css_post_types', $post_types );
+		$provider_post_types = array();
+		// Generate a name => name array for backwards compatibility.
+		foreach ( $post_types as $post_type ) {
+			$provider_post_types[ $post_type->name ] = $post_type->name;
+		}
+
+		return apply_filters(
+			'jetpack_boost_critical_css_post_types_archives',
+			// Keep this for backwards compatibility.
```

Comment:
Maybe a docblock with `@deprecated` is necessary? Or use https://developer.wordpress.org/reference/functions/apply_filters_deprecated/

Reply:
Thanks! Updated in 3198ec8.